### ISSUE TITLE
Port to JupyterLab

### ISSUE THREAD

**Issue by redeemefy** [Calysto/notebook-extensions#16]
Any plans to include this project to Jupyter Lab?

**Comment by dsblank** [comment]
Yes, absolutely! I haven't really started looking at JupyterLab yet, so if you would like to make a PR, or provide pointers on what needs to be done, that would be quite useful.

**Comment by redeemefy** [comment]
I ran the command `jupyter labextension install <foo>`, based on their documentation to install in `jupyterlab` just to see what it does. As expected didn't install. I went to their website and I found some documentation [here](http://jupyterlab-tutorial.readthedocs.io/en/latest/extensions_dev.html).
